clients/drcachesim/scheduler/scheduler_impl.cpp

```diff
@@ -161,7 +161,9 @@ scheduler_impl_tmpl_t<memref_t, reader_t>::get_reader(const std::string &path,
             // Skip the auxiliary files.
             if (fname == DRMEMTRACE_MODULE_LIST_FILENAME ||
                 fname == DRMEMTRACE_FUNCTION_LIST_FILENAME ||
-                fname == DRMEMTRACE_ENCODING_FILENAME || fname == DRMEMTRACE_V2P_FILENAME)
+                fname == DRMEMTRACE_ENCODING_FILENAME ||
+                fname == DRMEMTRACE_V2P_FILENAME ||
+                fname == DRMEMTRACE_TRACE_INFO_FILENAME)
```

Comment:
As noted, let's switch to an allowlist of known compression suffixes instead?  Or look for "drmemtrace" in the file name?

Reply:
A separate dir seems cleaner (see #7430 ).